ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml

```diff
@@ -26,31 +26,31 @@ images:
   to: generatetasks
 - context_dir: modules/create-vm/
   dockerfile_path: build/create-vm/Dockerfile
-  from: openshift_release_golang-1.15
+  from: ubi_minimal
   to: tekton-task-create-vm
 - context_dir: modules/copy-template/
   dockerfile_path: build/copy-template/Dockerfile
-  from: openshift_release_golang-1.15
+  from: ubi_minimal
   to: tekton-task-copy-template
 - context_dir: modules/execute-in-vm/
   dockerfile_path: build/execute-in-vm/Dockerfile
-  from: openshift_release_golang-1.15
+  from: ubi
   to: tekton-task-execute-in-vm
 - context_dir: modules/generate-ssh-keys/
   dockerfile_path: build/generate-ssh-keys/Dockerfile
   from: openshift_release_golang-1.15
   to: tekton-task-generate-ssh-keys
 - context_dir: modules/disk-virt-customize/
   dockerfile_path: build/disk-virt-customize/Dockerfile
-  from: openshift_release_golang-1.15
+  from: ubi_minimal
   to: tekton-task-disk-virt-customize
 - context_dir: modules/disk-virt-sysprep/
   dockerfile_path: build/disk-virt-sysprep/Dockerfile
-  from: openshift_release_golang-1.15
+  from: ubi_minimal
```

Comment:
we are getting `/bin/sh: find: command not found` for these 2.

One issue I noticed is that the CI sometimes replaces the images we provide with a custom ones (even different versions).

Can you debug this?
- check if you can locate /bin/find
- check if PATH is setup correctly
- maybe we need to install `findutils` ourselves in the CI?